video/vdu_sys.h

```diff
@@ -86,6 +91,9 @@ void VDUStreamProcessor::vdu_sys() {
 			case 0x1C: {					// VDU 23, 28
 				vdu_sys_hexload();
 			}	break;
+			case 0x1D: {					// VDU 23, 29
+				vdu_sys_updater();
+			}	break;
```

Comment:
my inclination would be to use a command inside VDU 23, 0 - commands in that range are intended for VDP programming, which feels like the best fit for this functionality.

maybe `VDU 23, 0, &A1` so it sits right next to the buffered command API?

VDU 23, 28-31 are historically _supposed_ to be "reserved for user calls".  we don't currently have a mechanism for facilitating that...

had I noticed that hexload was using 23,28 earlier I'd have probably pushed for that to move too - and likely will still do so :grin: